src/Compilers/CSharp/Test/Symbol/Symbols/Source/FieldTests.cs

```diff
@@ -563,6 +563,27 @@ struct S
     private static string s1 = $"""";
     private static readonly string s2 = $"""";
 }
+");
+
+            comp.VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void UnreferencedRawInterpolatedStringConstants()
```

Comment:
nit: I'd have moved the new test to the raw string literal test file (keeping same test/method name as original)

Reply:
hrmm... it's unclear the organization here.  i've been trying to keep things as siblings.  but i can move if you'd like :)

Reply:
That's fine (just a nit)